doozer/doozerlib/distgit.py

```diff
@@ -1286,6 +1289,61 @@ def update_build_db(self, success_flag, task_id=None, scratch=False):
                         traceback.print_exc()
                         self.logger.error(f'Unable to extract brew task information for {task_id}')
 
+    def get_installed_packages(self, image_pullspec) -> list:
+        bbii = BrewBuildImageInspector(self.runtime, image_pullspec)
+        installed_packages_dict = bbii.get_all_installed_package_build_dicts()
+        return [p['nvr'] for p in installed_packages_dict.values()]
+
+    def update_konflux_db(self, build_info, success, job_url='', scratch=False):
+        if scratch:
+            return
+
+        if not self.runtime.konflux_db:
+            self.logger.warning('Konflux DB connection is not initialized, not writing build record to the Konflux DB.')
+            return
+
+        image_pullspec = build_info['extra']['image']['index']['pull'][0]
+
+        self.logger.info('Storing build info in Konflux for Brew build %s', build_info['nvr'])
+        try:
+            source_repo, commitish = build_info['source'].split('#')
+
+            try:
+                release_target = self.org_release.split('.')[-1]
+            except KeyError:
+                release_target = ''
+            el_target = release_target if release_target else f'el{self.metadata.branch_el_target()}'
+
+            build_record = KonfluxBuildRecord(
+                name=self.metadata.name,
+                group=self.runtime.group,
+                version=build_info['version'],
+                release=build_info['release'],
+                assembly=self.runtime.assembly,
+                el_target=el_target,
+                arches=self.metadata.get_arches(),
+                installed_packages=self.get_installed_packages(image_pullspec),
+                parent_images=[build['nvr'] for build in build_info['extra']['image']['parent_image_builds'].values()],
+                source_repo=source_repo,
+                commitish=commitish,
+                embargoed='p1' in build_info['release'].split('.'),
+                start_time=datetime.strptime(build_info['start_time'], '%Y-%m-%d %H:%M:%S.%f'),
+                end_time=datetime.strptime(build_info['completion_time'], '%Y-%m-%d %H:%M:%S.%f'),
+                artifact_type=ArtifactType.IMAGE,
+                engine=Engine.BREW,
+                image_tag=image_pullspec,
```

Comment:
This is not being populated with a tag, but a full pullspec. 
I'm fine with storing that, but make an `image_pullspec` column. 
If you can image_tag to image_sha, you can then extract the `sha256:*` portion of the  pull spec.